miniapp/miniapp.cpp

```diff
@@ -50,6 +51,9 @@ int main(int argc, char** argv) {
     nest::mc::communication::global_policy_guard global_guard(argc, argv);
 
     try {
+        nest::mc::util::meter_manager meters;
+        meters.checkpoint("start");
```

Comment:
No, we don't. The current implementation just measures the time spent in the different parts of the time stepping.
This is higher level: providing time spent in initialization, model setup, model simulation, and shutdown.